source/1_engine/rex_engine/include/rex_engine/diagnostics/win/win_call.h

```diff
@@ -34,41 +37,50 @@ namespace rex
 
     private:
       DWord m_error;
+      DWord m_error_success;
```

Comment:
I don't see a reason why. strong types are mostly used for arguments, not for storing data

Reply:
To make it obvious for the reader of the header.

Reply:
make what more obvious?
strong types are not meant to be used as members, only as arguments.

Reply:
Why not, it's not because you don't use them as a member they should not be used as a member.
I do it all the time.
If you have a function setErrorCode(ErrorCode c) and a member s32 m_error_code. 
It might be confusing that this suddenly changed type.